src/actions/WalletListMenuActions.js

```diff
@@ -89,10 +91,36 @@ export function walletListMenuAction(walletId: string, option: WalletListMenuKey
       return (dispatch: Dispatch, getState: GetState) => {
         const state = getState()
         const { currencyWallets } = state.core.account
-        const wallet = currencyWallets[walletId]
-        const xPub = wallet.displayPublicSeed
-        const xPubExplorer = wallet.currencyInfo.xpubExplorer && xPub ? sprintf(wallet.currencyInfo.xpubExplorer, xPub) : ''
-        dispatch({ type: 'OPEN_VIEWXPUB_WALLET_MODAL', data: { xPub, walletId, xPubExplorer } })
+        const { displayPublicSeed, currencyInfo } = currencyWallets[walletId]
```

Comment:
Default displayPublicSeed to ''

Reply:
Derp, defaults don't work with `null`. This is an API design mistake in the core - we should have made it `| void` so defaults would work, instead of `| null` where defaults don't work.